bot/src/player/mod.rs

```diff
@@ -121,8 +121,11 @@ pub(self) async fn convert_item(
                 return Ok(None);
             }
 
+            // TODO: cache this value
+            let streamer: PrivateUser = spotify.me().await?;
```

Comment:
This should probably be passed in as an argument to `convert_item`.

`convert_item` is called in a pretty tight loop here for example, and it would result in a lot of extra API calls otherwise:
https://github.com/udoprog/OxidizeBot/blob/c008e003a48935656de6e2cbe840a9147d2981c6/bot/src/player/mixer.rs#L43